packages/ketcher-react/src/script/editor/operations/sgroup/index.ts

```diff
@@ -59,10 +59,12 @@ class SGroupCreate extends BaseOperation {
 
 class SGroupDelete extends BaseOperation {
   data: Data
+  checkEmptiness: boolean
 
-  constructor(sgroupId?: any) {
+  constructor(sgroupId?: any, checkEmptiness = true) {
```

Comment:
please rename to 'validate' instead

Reply:
Naming booleans starting with "is" considered a good practice. In this case "isValid" or "isEmpty" would be self-descriptive.